compiler/frontend/src/ast.rs

```diff
@@ -110,6 +110,7 @@ pub struct Function {
 pub struct Call {
     pub receiver: Box<Ast>,
     pub arguments: Vec<Ast>,
+    pub pipe: bool,
```

Comment:
When reading the name elsewhere in code, this makes it clearer that it doesn't store a representation of the pipe, but only whether a pipe was used or not. At least for Dart, there exists an official recommendation: [PREFER starting a boolean variable or property comment with “Whether” followed by a noun or gerund phrase](https://dart.dev/effective-dart/documentation#prefer-starting-a-boolean-variable-or-property-comment-with-whether-followed-by-a-noun-or-gerund-phrase)

```suggestion
    pub is_pipe: bool,
```